### PR TITLE
Feature/design iteration

### PR DESCRIPTION
**Pull request by b2l**
Design iteration, result of a day of work with @Joelhelin.
Changes:
* more doc (forms, variables)
* more variables for spacing (designer will try to make space a factor of 10px)
* adjust some colors
* restore the form elements
* new font: Source Sans Pro

### CODE REVIEW

File: doc_assets/styleguide.scss [hunk 1]
```diff
@@ -91,6 +91,14 @@ body{
   background: $white;
   border: 1px solid black;
 }
+.brand-primary-custom {
+  background: $brand-primary-custom;
+  border: 1px solid black;
```
Comment (mnd-hound) [inline doc_assets/styleguide.scss]: Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: doc_assets/styleguide.scss [hunk 2]
```diff
@@ -91,6 +91,14 @@ body{
...
+}
+.blue-light {
+  background: $blue-light;
+  border: 1px solid black;
```
Comment (mnd-hound) [inline doc_assets/styleguide.scss]: Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: doc_assets/styleguide.scss [hunk 3]
```diff
@@ -91,6 +91,14 @@ body{
   background: $white;
   border: 1px solid black;
 }
+.brand-primary-custom {
+  background: $brand-primary-custom;
+  border: 1px solid black;
+}
```
Comment (mnd-hound) [inline doc_assets/styleguide.scss]: Rule declaration should be followed by an empty line

File: doc_assets/styleguide.scss [hunk 4]
```diff
@@ -91,6 +91,14 @@ body{
...
+.blue-light {
+  background: $blue-light;
+  border: 1px solid black;
+}
```
Comment (mnd-hound) [inline doc_assets/styleguide.scss]: Rule declaration should be followed by an empty line

File: src/mnd-bootstrap/css/_forms.scss [hunk 5]
```diff
@@ -91,14 +120,13 @@ category: CSS
...
-  font-size: $font-size-small;
-  margin: 0;
-  text-transform: uppercase;
+  color: $gray;
+  margin-bottom: $space;
 }
 
 .checkbox label,
 .radio label {
+  margin-bottom: 0;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/css/_forms.scss]: Properties should be ordered color, font-size, margin-bottom, text-transform

File: src/mnd-bootstrap/_variables.scss [hunk 6]
```diff
@@ -87,7 +87,8 @@ $link-hover-color:      darken($link-color, 15%);
 //
 //## Font, line-height, and color for body text, headings, and more.
 
-$font-family-sans-serif:  "Helvetica Neue", Arial, sans-serif;
+@import url(http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,300,400italic,600,600italic,300italic);
```
Comment (jorkas) [inline src/mnd-bootstrap/_variables.scss]: Maybe we can add some easier way to read? Like: 300,400,600,300italic,400italic,600italic

Comment (b2l) [inline src/mnd-bootstrap/_variables.scss]: I think we should not use google font at all, at least for performance reason.
But I don't know how to expose fonts with the current setup

**Comment by eliashedlund** [comment]
There are a few places where we use #fff and others where we use $white.
Maybe $white is a bit overkill? But I like it. We should at least be consistent.

**Pull request merged**